ios/FluentUI/Core/SwiftUI+ViewModifiers.swift

```diff
@@ -52,12 +52,29 @@ extension View {
     }
 
     /// Applies multiple shadows on a View
-    /// - Parameters
+    /// - Parameters:
     ///  - shadowInfo: The values of the two shadows to be applied
     /// - Returns: The modified view.
     func applyShadow(shadowInfo: ShadowInfo) -> some View {
         modifier(ShadowModifier(shadowInfo: shadowInfo))
     }
+
+    /// Abstracts away differences in pre-iOS 17 `onChange(of:perform:)` versus post-iOS 17 `onChange(of:_:)`.
+    ///
+    /// This function will be removed once FluentUI moves to iOS 17 as a minimum target.
```

Comment:
Do we have a bug tracking this?

Reply:
I think that any time we bumped minimum OS versions in the past, the bumper would also search the codebase for any relevant availability checks -- the `if #available(iOS 17, *)` should trip that search query when we get there. But let's chat more offline to see if there's something we can do to better formalize this process.